pheweb/serve/server.py

```diff
@@ -235,12 +243,14 @@ def api_gene_functional_variants(gene):
     if ('p' in request.args):
         pThreshold= float(request.args.get('p'))
     annotations = jeeves.gene_functional_variants(gene, pThreshold)
+    for anno in annotations:
+        anno['significant_phenos'] = [pheno for pheno in anno['significant_phenos'] if pheno.phenocode in use_phenos]
     return jsonify(annotations)
 
 @app.route('/api/lof')
 @check_auth
 def api_lof():
-    lofs = jeeves.get_all_lofs(conf.lof_threshold)
+    lofs = [lof for lof in jeeves.get_all_lofs(conf.lof_threshold) if lof['gene_data']['pheno'] in use_phenos]
```

Comment:
maybe jeeves should do the filtering so this repetition is not bloating server

Reply:
Agreed in principle - decided for this way as we need to filter in server anyway and found it cleaner to filter in one of them only

Reply:
is there other filtering in server than where get_best_phenos_by_gene is called? If it is then for sure only in one place is better.
REgardless get_best_phenos_by_gene should move to jeeves or db so the concern of how to fetch data is not on server

Reply:
Yes now all filtering is done in server, there's also server specific stuff like autocomplete. Yup now the function is in jeeves